src/contracts/addreses.ts

```diff
@@ -199,31 +199,31 @@ const tokens: Record<GebDeployment, TokenList> = {
         },
         WSTETH: {
             address: '0x5979D7b546E38E414F7E9822514be443A4800529',
-            collateralJoin: '0x0',
-            collateralAuctionHouse: '0x0',
+            collateralJoin: '0xae7Df58bB63b2Db798f85AB7BCACE340d55f6f39',
+            collateralAuctionHouse: '0x0365dFC776851e970bd6269a2862eFc9a6265273',
             decimals: 18,
             symbol: 'WSTETH',
             bytes32String: WSTETH,
             isCollateral: true,
         },
         RETH: {
             address: '0xEC70Dcb4A1EFa46b8F2D97C310C9c4790ba5ffA8',
-            collateralJoin: '0x0',
-            collateralAuctionHouse: '0x0',
+            collateralJoin: '0xC215F3509AFbB303Bf4a20CBFAA5382fad9bEA1D',
+            collateralAuctionHouse: '0x51a423B43101B219a9ECdEC67525896d856186Ec',
             decimals: 3,
```

Comment:
RETH has 18 decimals, if I am understanding this correctly:
https://arbiscan.io/token/0xec70dcb4a1efa46b8f2d97c310c9c4790ba5ffa8